.github/workflows/smoke-tests.yml

```diff
@@ -61,6 +61,8 @@ jobs:
       - name: Compile dd-trace-go
         run: go build $PACKAGES
       - name: Test dd-trace-go
+        env:
+          DD_APPSEC_WAF_TIMEOUT: 1h
```

Comment:
"Large enough":

![Huuuge](https://mcvuk.com/wp-content/uploads/Huuuge_LOGO_CZERWONE_RGB.png)

Reply:
I'd prefer 0 as disable TBH, so I went with "infinite" instead ^^
Maybe worth considering in the future 0 as a disablement mechanism for our CIs.